crates/sui-benchmark/src/workloads/adversarial.rs

```diff
@@ -58,6 +71,13 @@ impl std::fmt::Display for AdversarialTestPayload {
 
 impl Payload for AdversarialTestPayload {
     fn make_new_payload(&mut self, effects: &ExecutionEffects) {
+        // // Sometimes useful when figuring out why things failed
```

Comment:
Let's clean this up before landing? This printing can be added to the "Adversarial transactions should never abort" message if it's helpful.